src/EnergyPlus/CMakeLists.txt

```diff
@@ -645,6 +664,16 @@ add_subdirectory(AirflowNetwork)
 #message( $<TARGET_PROPERTY:airflownetworklib,INTERFACE_INCLUDE_DIRECTORIES> )
 #include_directories($<TARGET_PROPERTY:airflownetworklib,INTERFACE_INCLUDE_DIRECTORIES>)
 
+add_library( energyplusparser STATIC ${INPUTPARSING_SRC} )
+add_dependencies( energyplusparser GenerateEmbeddedEpJSONSchema )
+target_link_libraries( energyplusparser re2 )
+if(UNIX AND NOT APPLE)
+  target_link_libraries( energyplusparser dl )
+endif()
+if (WIN32)
+  target_link_libraries( energyplusparser Shlwapi )
+endif()
```

Comment:
Hmm I expected E+ to depend on this DLL.  It's fine to continue building E+ as-is with the input processing built into it, I just sorta expected the E+ DLL to depend on this.  The code isn't duplicated anywhere, just some extra build rules, and negligible extra build time, so all good.

Reply:
E+ could depend on this library. I don't have strong feelings one way or another.

Reply:
I don't think we should make that change, not yet anyway.